features/login/impl/src/main/kotlin/io/element/android/features/login/impl/screens/loginpassword/LoginPasswordView.kt

```diff
@@ -108,48 +117,46 @@ fun LoginPasswordView(
             )
         }
     ) { padding ->
-        Box(
+        val scrollState = rememberScrollState()
+
+        Column(
             modifier = Modifier
                 .fillMaxSize()
                 .imePadding()
                 .padding(padding)
                 .consumeWindowInsets(padding)
+                .verticalScroll(state = scrollState)
+                .padding(horizontal = 16.dp),
         ) {
-            val scrollState = rememberScrollState()
-
-            Column(
+            // Title
+            IconTitleSubtitleMolecule(
+                modifier = Modifier.padding(top = 20.dp, start = 16.dp, end = 16.dp),
+                iconImageVector = Icons.Filled.AccountCircle,
+                title = stringResource(
+                    id = R.string.screen_account_provider_signin_title,
+                    state.accountProvider.title
+                ),
+                subTitle = stringResource(id = R.string.screen_server_confirmation_message_login_matrix_dot_org)
+            )
+            Spacer(Modifier.height(40.dp))
+            LoginForm(
+                state = state,
+                isLoading = isLoading,
+                onSubmit = ::submit
+            )
+            Spacer(Modifier.height(24.dp))
+            Spacer(modifier = Modifier.weight(1f))
+            // Submit
+            ButtonWithProgress(
+                text = stringResource(R.string.screen_login_submit),
+                showProgress = isLoading,
+                onClick = ::submit,
+                enabled = state.submitEnabled || isLoading,
```

Comment:
This may be changed here as well, WDYT?

https://github.com/vector-im/element-x-android/blob/develop/features/login/impl/src/main/kotlin/io/element/android/features/login/impl/screens/confirmaccountprovider/ConfirmAccountProviderView.kt#L94